introduction.md

```diff
@@ -1,2 +1,10 @@
 Hello, this is me. I'm in California, dreaming about who we used to be...
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?